include/vinecopulib/misc/implementation/tools_interpolation.ipp

```diff
@@ -190,7 +190,7 @@ InterpolationGrid::intergrate_1d(const Eigen::MatrixXd &u,
             tmpgrid.col(0) = grid_points_;
             tmpgrid.col(1) = Eigen::VectorXd::Constant(m, u2);
         }
-        tmpvals = interpolate(tmpgrid);
+        tmpvals = interpolate(tmpgrid).array().max(1e-4);
```

Comment:
1e-4 is rather "large" no?

Reply:
yes and no.. the integral over [0, 1] would be just 1e-4 which is rather small compared to the actual value of the integral (1). I came up with this value in a few experiments (nothing too solid) that showed it shouldn't be less than `1e-5` because estimates get highly instable (set to `1e-10`) below that.

Reply:
I see, then it's fine. Any idea why the windows check fail?

Reply:
`there is no package called 'mvtnorm'`
I blame ALTREP as always ;) But what confuses me is that the checks run for 1hr, they should be done after ~30mns.